network_models/xray_flux_forecast/starting_point/network.py

```diff
@@ -84,16 +79,24 @@
     x = concatenate(input_images)
 else:
     x = input_images[0]
-x = Conv2D(1, (1,1), strides=(1,1), padding='same', activation="relu")(x)
-x = MaxPooling2D(pool_size=(1024, 1024), strides=(1,1), padding='valid')(x)
+    
+x = Conv2D(16, kernel_size=16, padding = 'same', activation='relu', use_bias=False, kernel_initializer = 'lecun_uniform')(x)
+x = Conv2D(16, kernel_size=16, padding = 'same', activation='relu', use_bias=False, kernel_initializer = 'lecun_uniform')(x)
```

Comment:
It looks like you brought in a bunch of network changes accidentally. If you are doing this from the command line, try `git add -p network_models/xray_flux_forecast/starting_point/network.py` next time.